### PR TITLE
Adds solution

### PR DESCRIPTION
**Pull request by ezmiller**
@pletcher @AnnJohn opening for initial review.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    .catch((err) => {
+      return done(err);
+    });
+}));
+
+passport.serializeUser(function(user, done) {
+  done(null, user.id);
+});
+
+passport.deserializeUser(function(user, done) {
```
Comment (pletcher) [inline app/index.js:70]: These functions are the worst parts of passport (I think, anyway), but I really like how you've explained and used them.

**Comment by pletcher** [comment]
The solution looks great! Once `wip-master` is ready, let's just update the README here with the README updates there and we'll be good to go!

File: app/index.js (hunk 1 above)
Comment (ezmiller) [inline app/index.js:70]: I know! What horrible fancy-pants naming. Worst of engineering.

**Comment by ezmiller** [comment]
@pletcher I added the readme.

**Comment by pletcher** [comment]
:shipit:

**Pull request merged**